src/v3/test/integration/authenticator-enroll-email-magic-link-false.test.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import mockResponse from '@okta/mocks/data/idp/idx/authenticator-enroll-email-emailmagiclink-false.json';
+import { createAuthJsPayloadArgs, setup } from './util';
+
+describe('authenticator-enroll-email-magic-link-false', () => {
```

Comment:
I think better descriptions in these tests would be helpful in case they fail to understand what we're testing. e.g.here `authenticator-enroll when email magic link = false` or something like that? Same with other integration tests.